README.md

```diff
@@ -70,3 +70,28 @@ A facsimile of `openSUSE:Factory` in the form of a subset of the related data ca
 Some tests will attempt to run against the local OBS, but not all.
 
     nosetests
+
+## Running Continuous Integration
+
+This repository includes all the needed files to set up and run the Continuous Integration test suite. The idea is to use Docker Compose to orchestrate a set of containers, including an OBS instance, and run [the tests](tests/) on top of them. Although they automatically run [on GitHub Actions](https://github.com/features/actions) (more on that later), it is easy to run them locally. The following commands must be executed from the root of the repository.
+
+
+    # Mount the current path at the /code directory on the container
+    sed -i -e "s,../..:,$PWD:," dist/ci/docker-compose.yml
+
+    # Run the linter
+    docker-compose -f dist/ci/docker-compose.yml run flaker
+
+    # Run the test suite. It may take some time.
+    docker-compose -f dist/ci/docker-compose.yml run test
```

Comment:
BTW: to run one test only, you need the quite obvious command line:
`docker-compose -f dist/ci/docker-compose.yml run test run_as_tester nosetests tests/util_tests.py`

(it's a test after all, so test shouldn't be less than 5 times in the command line :smile:)

Reply:
Oh, pretty obvious :smile: I have added the example to the README. Thanks!